msteams-platform/bots/how-to/bot-configuration-experience.md

```diff
@@ -11,18 +11,27 @@ ms.localizationpriority: high
 
 > [!NOTE]
 >
-> * Bot configuration experience is available in [public developer preview](../../resources/dev-preview/developer-preview-intro.md).
-> * Bot configuration experience is supported in channel or group chat scopes only.
+> Bot configuration experience is supported in channel or group chat scopes only.
 
-You can create a bot to enable the bot configuration settings for the user during the bot installation and also from the channel or group chat scope after the bot is installed.
+When you're creating a bot for Teams, it’s important to make sure users have a smooth experience and provision control to personalize their bots in places like Microsoft Teams. The ability to configure bots post-installation within Teams allows developers to fine-tune bot functionality according to user requirements even after deployment. You can use methods from the Microsoft Bot Framework SDK to implement bot configuration features that enhance user engagement and productivity. 
+ 
+In this article, we'll explore methods for configuring bots post-installation, both for cross-platform compatibility and specific to Teams. You can create complex bot configurations that enhance user engagement and efficiency.
 
```

Comment:
also add a snippet on how bot configuration is initiated while installing bot in said scope, if opted in